doc/book/admin/upgrades/1.6-2.0-downtime.rst

```diff
@@ -0,0 +1,20 @@
+..  _admin-upgrades-1.6-2.x_downtime:
+
+Upgrade from 1.6 directly to 2.x
```

Comment:
May to  rename to 

Upgrade from 1.6 directly to 2.x with downtime

to make it clear what the case is (it this is the case, of course )))